drake/doc/developers.rst

```diff
@@ -8,31 +8,78 @@ The standard github workflow is to fork the drake repository into your
+github website.  Please read on for information on testing, code
+review, and code style.
 
 We would like to hear about your success stories if you've used
 Drake in your own projects.  Please consider contributing to our :doc:`gallery`
 by editing ``gallery.rst`` in the ``drake/doc/`` directory and submitting a pull
 request.
 
+Licensing
+=========
+
 **Important note:** Drake is an open source project licensed under
 extremely flexible terms intended to encourage use by anyone, for any
 purpose. When you make a contribution to the Drake project, you are
 agreeing to do so under those same terms.
 
+Testing
+=======
+
+When you create a pull request, unit and regression tests will
+automatically run on a number of build servers.  You can run those
+tests locally by running ``make test`` from the command line. Note
+that there are a lot of computationally demanding tests and this could
+run for a few hours depending on your machine.
+
+Your change must include unit tests that protect it against regressions,
+and those tests must pass on all platforms supported by Drake.
+
+**For C++ changes,** please use the googletest framework. It is already
+available in the superbuild.
+
+**For MATLAB changes,** please write an example. Every example and static
+class method runs as a unit test, and fails if it times out or calls
+``error``. For more information, see ``drake/util/unitTest.m``.
+
+Code Review
+===========
+
+For complex changes, especially those that will span multiple PRs, please
+open a GitHub issue and solicit design feedback before you invest a lot of
+time in code.
+
+Be prepared to engage in active code review on your pull requests.  The Drake
+code review process has two phases: feature review and platform review. You
+are responsible for finding reviewers, and for providing them the information
+they need to review your change effectively. If a reviewer asks you for more
+information, that is a sign you should add more documentation to your PR. 
+
+**Feature Review.** After creating your pull request, assign it to someone
+else on your team for feature review. Choose the person most familiar
+with the context of your pull request. This reviewer is responsible for
+protecting your team by inspecting for bugs, for test coverage, and for 
+alignment with the team's goals. During this review, you and your reviewer
+should also strive to minimize the number of changes that will be necessary
+in platform review.
+
+**Platform Review.** After your feature reviewer has signed off on your change,
+reassign it to a Drake owner for platform review. The owner will inspect for
+architectural compatibility, stability, performance, test coverage, and style.
+Once satisfied, the owner merges the PR. 
+
+The following github users are Drake owners. If possible, seek platform review 
+from an owner who has previously reviewed related changes. Shared context will 
+make the review faster! @RussTedrake is the creator of Drake, but has limited
```

Comment:
while i appreciate the concern, i'd like to remove the line about how busy i am.  :)